apps/style/javaIde/style.scss

```diff
@@ -0,0 +1,16 @@
+.container {
```

Comment:
I'd also suggest renaming this file to be more unique i.e. javaIdeStyle.scss or similar.

Reply:
our other style files are all just named `style`, it's in a javaIde folder to differentiate it

Reply:
👍